spec/level_db_spec.rb

```diff
@@ -65,6 +65,62 @@
       db.close
     end
 
+    describe '#compact_range' do
+      let :leveldb do
+        leveldb = double(:db)
+        leveldb.stub(:compact_range) do |*args|
+          calls << args.map { |arg| arg && String.from_java_bytes(arg) }
+        end
+        leveldb
+      end
+
+      let :calls do
+        []
+      end
+
+      it 'compacts the given range' do
+        LevelDb::Db.new(leveldb).compact_range(from: 'one', to: 'three')
+        calls.should include(['one', 'three'])
+      end
+
+      it "compacts the whole range to the 'to' key when the 'from' option is omitted" do
```

Comment:
Why not `it 'compacts … "to" … "from" … '`? (I'm thinking of quoting consistency).